app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,19 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Tasky"
-        tools:targetApi="31" />
+        tools:targetApi="31">
+        <activity
+            android:name=".BaseActivity"
+            android:exported="false" />
```

Comment:
Since we removed base activity, we should remove it from here also

Reply:
Terrible mistake !! 

Done